Source/WebCore/html/HTMLScriptElement.h

```diff
@@ -36,9 +39,15 @@ class HTMLScriptElement final : public HTMLElement, public ScriptElement {
     static Ref<HTMLScriptElement> create(const QualifiedName&, Document&, bool wasInsertedByParser, bool alreadyStarted = false);
 
     String text() const { return scriptContent(); }
-    WEBCORE_EXPORT void setText(String&&);
+    WEBCORE_EXPORT void setTextExported(String&&);
```

Comment:
Not sure what to call this function?

I couldn't leave it as setText else I got a build error. The ambiguity I think is from converting NSString* to an appropriate type, not sure if there's a better way to avoid the ambiguity?

```
In file included from /Users/luke/Development/BrowserEngines/webkit/WebKitBuild/Debug/DerivedSources/WebKitLegacy/unified-sources/UnifiedSource11-mm.mm:3:
/Users/luke/Development/BrowserEngines/WebKit/Source/WebKitLegacy/mac/DOM/DOMHTMLScriptElement.mm:52:11: error: call to member function 'setText' is
      ambiguous
    IMPL->setText(newText);
    ~~~~~~^~~~~~~
In file included from /Users/luke/Development/BrowserEngines/webkit/WebKitBuild/Debug/DerivedSources/WebKitLegacy/unified-sources/UnifiedSource11-mm.mm:3:
In file included from /Users/luke/Development/BrowserEngines/WebKit/Source/WebKitLegacy/mac/DOM/DOMHTMLScriptElement.mm:32:
/Users/luke/Development/BrowserEngines/webkit/WebKitBuild/Debug/WebCore.framework/PrivateHeaders/HTMLScriptElement.h:42:25: note: candidate function
    WEBCORE_EXPORT void setText(String&&);
                        ^
/Users/luke/Development/BrowserEngines/webkit/WebKitBuild/Debug/WebCore.framework/PrivateHeaders/HTMLScriptElement.h:43:23: note: candidate function
    ExceptionOr<void> setText(std::variant<RefPtr<TrustedScript>, String>&&);
```

Reply:
A fix to the ambiguity is to write `IMPL->setText(String(newText))`.